docs/blockchain/contracts/nft-contract/nft-tables.experimental.md

```diff
@@ -0,0 +1,812 @@
+| -------------------- | -------------------- | -------------------------------------------------------------------------------------------- |
+| offer_id             | uint64_t             | Uniq offer ID                                                                                |
+| buyer                | name                 | Buyer who makes the offer for the Uniq                                                       |
+| receiver             | std::optional\<name> | Users who will receive the Uniq if offer is accepted, if specified                           |
+| price                | asset                | Offer price in UOS                                                                           |
+| promoter_basis_point | uint16_t             | UOS share received by the promoter with this offer. Specified in basis points. 1 means 0.01% |
+| expiry_date          | time_point_sec       | Expiry date of the offer in seconds from the EPOCH                                           |
+
+Most relevant actions: `mknftofr.a`
+
+-   `cleos` Query Example
+
+```sh
+cleos get table eosio.nft.ft <UNIQ ID> nftoffer.a
+```
+
+-   `curl` query example
+
+```sh
+curl <NODEOS_API_IP>/v1/chain/get_table_rows -X POST -d '{"scope":"<UNIQ ID>", "code":"eosio.nft.ft", "table":"nftoffer.a", "json": true}'
+```
+
+## next.fctofr
+
+-   Table: `next.fctofr`
+-   Code: `eosio.nft.ft`
+-   Scope: `0`
+-   Key: `N/A`
+
+The table stores information about the ID of the next created Uniq offer
+
+| Fields | Type     | Description                                              |
+| ------ | -------- | -------------------------------------------------------- |
+| value  | uint64_t | ID that the next created Uniq factory offer will receive |
+
+Most relevant actions: `mkfctofr.a`
+
+-   `cleos` Query Example
+
+```sh
+cleos get table eosio.nft.ft 0 next.fctofr
+```
+
+-   `curl` query example
+
+```sh
+curl <NODEOS_API_IP>/v1/chain/get_table_rows -X POST -d '{"scope":"0", "code":"eosio.nft.ft", "table":"next.fctofr", "json": true}'
+```
+
+## fctoffer.a
+
+-   Table: `fctoffer.a`
+-   Code: `eosio.nft.ft`
+-   Scope: `Uniq factory ID`
+-   Key: `id`
+
+The table stores information about the offer made by the buyer for specific Uniq factory ID
+
+| Fields               | Type                 | Description                                                                                  |
+| -------------------- | -------------------- | -------------------------------------------------------------------------------------------- |
+| offer_id             | uint64_t             | Uniq factory offer ID                                                                        |
+| buyer                | name                 | Buyer who makes the offer for Uniq factory                                                   |
+| receiver             | std::optional\<name> | Users who will receive the Uniq if offer is accepted, if specified                           |
```

Comment:
Description should be
User who will receive ..

Reply:
updated